Source/NSKeyValueObserving.m

```diff
@@ -1713,10 +1713,11 @@ - (void) willChangeValueForKey: (NSString*)aKey
   pathInfo = [info lockReturningPathInfoForKey: aKey];
   if (pathInfo != nil)
     {
+      RETAIN(pathInfo);
       if (pathInfo->recursion++ == 0)
         {
           id    old = [pathInfo->change objectForKey: NSKeyValueChangeNewKey];
-
+          
```

Comment:
This tab should not be included and I am also not sure about the formatting change below.

Reply:
Fixed